src/components/Col.js

```diff
@@ -1,18 +1,19 @@
+import style from 'flexboxgrid';
```

Comment:
I find it' usually clearer to group the external imports separate from the internal ones, that's all.